File: aoj/itp1/topic4/4_c_simple_calculator.cc
#include <iostream>

int calc(char op, int a, int b) {
  switch (op) {
    case '+':
      return a + b;
    case '-':
      return a - b;
    case '*':
      return a * b;
    default:
      return a / b;
  }
}

int main() {
  while (true) {
    int a, b;
    char op;
    std::cin >> a >> op >> b;
    if (op == '?') return 0;

    std::cout << calc(op, a, b) << std::endl;
  }
}

File: aoj/itp1/topic4/4_c_simple_calculator.rb
def calc(operant, num_a, num_b)
  case operant
  when '+'
    num_a + num_b
  when '-'
    num_a - num_b
  when '*'
    num_a * num_b
  else
    num_a / num_b
  end
end

loop do
  a, op, b = gets.chomp.split
  break if op == '?'

  puts calc(op, a.to_i, b.to_i)
end
